core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -556,11 +556,14 @@ private static String toSql(RelNode root, SqlDialect dialect,
     final String bigQueryExpected = "SELECT '1' AS a, SKU + 1 AS B, '1' AS d\n"
         + "FROM foodmart.product\n"
         + "GROUP BY 1, B";
+    final String expectedSpark = "SELECT '1' a, SKU + 1 B, '1' dn"
+        + "FROM foodmart.productn"
```

Comment:
Is "productn" a typo; instead should it be "product \n"
Please check

Reply:
Made changes